libportal/filechooser.c

```diff
@@ -437,3 +443,100 @@ xdp_portal_save_file_finish (XdpPortal *portal,
   ret = g_task_propagate_pointer (G_TASK (result), error);
   return ret ? g_variant_ref (ret) : NULL;
 }
+
+/**
+ * xdp_portal_save_files:
+ * @portal: a #XdpPortal
+ * @parent: (nullable): parent window information
+ * @title: title for the file chooser dialog
+ * @modal: whether the dialog should be modal
+ * @current_name: (nullable): suggested filename
```

Comment:
The implementation doesn't use current_name, so we should drop it from the api here

Reply:
Hmm, I left this in the actual portal call since I figured applications might want to suggest a folder name, but I guess that probably doesn't make sense at least for GTK? I guess on that basis I should take it out of the portal impls as well?

Reply:
I don't think the gtk file chooser lets you suggest folder names, or does it ? If it does, sure, keep it in. But then the docs should be clearer that this is the suggested name for a newly created directory containing the files to save.